src/Common/RJSF/utils.tsx

```diff
@@ -135,3 +136,40 @@ export const getInferredTypeFromValueType = (value) => {
             return 'null'
     }
 }
+
+const conformPathToPointers = (path: string) => {
+    const trimmedPath = path.trim()
+    const isSlashSeparatedPathMissingBeginSlash = trimmedPath.match(/^\w+(\/\w+)*$/g)
+    if (isSlashSeparatedPathMissingBeginSlash) {
+        return `/${trimmedPath}`
+    }
+    const isDotSeparatedPath = trimmedPath.match(/^\w+(\.\w+)*$/g)
+    if (isDotSeparatedPath) {
+        // NOTE: replacing dots with forward slash (/)
+        return `/${trimmedPath.replaceAll(/\./g, '/')}`
+    }
+    return trimmedPath
+}
+
+export const parseSchemaHiddenType = (hiddenSchema: HiddenType): MetaHiddenType => {
+    if (!hiddenSchema) {
+        return null
+    }
+    const clone = structuredClone(hiddenSchema)
+    if (typeof clone === 'string') {
+        return {
+            value: false,
+            path: conformPathToPointers(clone),
+        }
+    }
+    if ('condition' in clone) {
```

Comment:
Are we sure we want to iterate over prototype chain or should we use hasOwnProperty?

Reply:
there is an eslint rule that prevents me from using hasOwnProperty & additionally suppose I use `Object.hasOwn(clone, 'condition')`, typescript can't see that it has that property therefore throwing an error.

Reply:
Okay as per our discussion using && before using in operator. Changed to -> `Object.hasOwn(clone, 'condition') && 'condition' in clone`